[All Hosts] (manifest) account for unified manifest in UX/UI articles (#4757)
OfficeDev/office-js-docs-pr: docs/includes/unified-manifest-support-note.md

> [!NOTE]

> > The [unified manifest for Microsoft 365](../develop/unified-manifest-overview.md) can be used in production Outlook add-ins. It is available only as a preview for Excel, PowerPoint, and Word add-ins. 

Reviewer: ```suggestion
> The [unified manifest for Microsoft 365](../develop/unified-manifest-overview.md) can be used in production Outlook add-ins. It's available only as a preview for Excel, PowerPoint, and Word add-ins. 
```
